docs/trace/building-your-own-exporter/Program.cs

```diff
@@ -24,12 +24,13 @@ public class Program
 
     public static void Main()
     {
-        using var tracerProvider = Sdk.CreateTracerProvider(
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddActivitySources(

```

Comment:
Trying to get more ideas - which one is more idiomatic? - `AddActivitySource` overload (singular form) or `AddActivitySources` (plural form).

Reply:
Should people know that what we did so far is based on Activity? What if in the future it changes to something else? Should we remove the name Activity from it?

Reply:
I feel like singular but with an overload for plural?
```csharp
void Add[Activity]Source(this TraceProviderBuilder builder, string sourceName);
void Add[Activity]Source(this TraceProviderBuilder builder, params string[] sourceNames);
```